README.md

```diff
@@ -36,8 +36,24 @@ The easiest way to build daal4py is using the conda-build with the provided reci
 * Python version >= 3.6
 * conda-build version >= 3
 * C++ compiler with C++11 support
+For oneAPI support:
+* A DPC++ compiler 
+* Intel(R) oneAPI Data Analytics Library version 2021.1 or later (https://github.com/01org/daal)
+  * You can use the pre-built conda package from Intel channel on anaconda.org (see below)
 
 ## Building daal4py
+For build with oneAPI support additional step requered:
+1. DPC++ compiler - DPCPPROOT variable should be defiend and pointed to DPCPP compier
+2. Intel(R) oneDAL with oneAPI support
+    a. Can be installed from Conda channel
+    b. Can be installed fron oneAPI packages repository and passed via DALROOT variable
+
+```
+export DPCPPROOT=/opt/intel/oneapi/compiler/latest
+export DALROOT=/opt/intel/oneapi/daal/latest
+```
```

Comment:
I'd split this code block and insert the relevant line after each item in the list above

Reply:
done